bird-slideshow.py

```diff
@@ -490,12 +477,18 @@ def update_img():
     canvas.create_image(
         (win_width)/2,
         (win_height)/2,
-        anchor = CENTER,
+        anchor = tkinter.CENTER,
         image = slideshow_imgs[imgs_index].tk_img
     )
 
 
 def next_img():
+    """Updates the imgs_index, then calls update_img() to put the image to the screen.
+    After wait_time, it calls itself again, recursively.
```

Comment:
This does not call itself recursively.  It schedules itself to be called it the future, by a thread of the Tk system.   So this doc string is not accurate